HLTriggerOffline/Btag/interface/HLTBTagHarvestingAnalyzer.h

```diff
@@ -48,6 +49,8 @@ class HLTBTagHarvestingAnalyzer : public DQMEDHarvester {
 			// Histogram handler
 			std::map<std::string, MonitorElement *> H1_;
 
+			std::vector<std::string> modules_;
```

Comment:
hi @natalia-korneeva - sorry i missed your answer - i was worried instead about modules_ in HLTBTagPerformanceAnalyzer::analyze